examples/inc_cylinder_2d/inc_cylinder_2d.ini

```diff
@@ -6,11 +6,11 @@ rank-allocator = linear
 ;cblas = Enter path to local BLAS library for OpenMP backend
 
 [constants]
-nu = 0.008
+nu = 0.005
```

Comment:
Do we need to change the documentation to reflect this?

Reply:
I don't think we mention the Reynolds number anywhere. However, the figure needs to be changed. 

I will update the documention. Do you want a separate commit or shall I push everything here and rename the commit.?

Reply:
A separate commit is fine.